Demo/StepperFilterDemoView.swift

```diff
@@ -23,7 +23,11 @@ class StepperFilterDemoView: UIView {
     override init(frame: CGRect) {
         super.init(frame: frame)
         addSubview(stepperFilterView)
-        stepperFilterView.fillInSuperview()
+        NSLayoutConstraint.activate([
+            stepperFilterView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            stepperFilterView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            stepperFilterView.bottomAnchor.constraint(equalTo: topAnchor, constant: 510 / 2),
```

Comment:
Magic number? 😄

Reply:
The idea was that the bottom sheet would have a height of at least 510. 
Any suggestions on the best way to get the view's height?

Reply:
Added a fix, but if you have a better way let me know! 😄